Activities/Cryptography/UiPath.Cryptography/CryptographyHelper.cs

```diff
@@ -222,7 +274,10 @@ private static SymmetricAlgorithm GetSymmetricAlgorithm(SymmetricAlgorithms symm
             switch (symmetricAlgorithm)
             {
                 case SymmetricAlgorithms.AES:
-                    return new AesCryptoServiceProvider(); // TODO: Use AesCng after upgrading to .NET Framework 4.6.2
+                    return new AesCryptoServiceProvider(); // kept for backwords compat
+                case SymmetricAlgorithms.AESGCM:
```

Comment:
Nitpick: Empty line before this

Also before the default below.

Reply:
added